pkg/jms-sdk-go/model/token.go

```diff
@@ -69,6 +69,8 @@ type ConnectTokenInfo struct {
 const (
 	ACLReview = "acl_review"
 	ACLReject = "acl_reject"
+
+	ACLFaceVerify = "acl_face_verify"
 )
 
 type ConnectOptions struct {
```

Comment:
The code seems to be correct and contains no apparent irregularities, issues or optimizations needed at this moment.